050_api_L214.py

```
@@ -0,0 +1,479 @@
+def face_detect(images):
+    detector = face_detection.FaceAlignment(face_detection.LandmarksType._2D, 
+                                            flip_input=False, device=device)
+    batch_size = args.face_det_batch_size
+    while True:
+        predictions = []
+        try:
+            for i in range(0, len(images), batch_size):
+                predictions.extend(detector.get_detections_for_batch(np.array(images[i:i + batch_size])))
+        except RuntimeError:
+            if batch_size == 1: 
+                raise RuntimeError('Image too big to run face detection on GPU. Please use the --resize_factor argument')
```

Ruff (0.14.5)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)